src/main/java/reposense/parser/CsvParser.java

```diff
@@ -51,105 +51,106 @@ public CsvParser(Path csvFilePath) throws IOException {
             }
         }
-
         return false;
     }
 
     /**
-     * Checks that {@code position} in within the range of {@code element} array and
-     * value in {@code position} is not empty.
+     * Returns the value of {@code record} at {@code colNum}.
      */
-    private boolean containsValueAtPosition(final String[] elements, int position) {
-        return elements.length > position && !elements[position].isEmpty();
+    protected String get(final CSVRecord record, int colNum) {
+        return record.get(colNum).trim();
     }
 
     /**
-     * Removes the override keyword for {@code position} in {@code elements}.
+     * Returns the value of {@code record} at {@code colNum} if present, or
+     * returns {@code defaultValue} otherwise.
      */
-    protected void removeOverrideKeywordFromElement(final String[] elements, int position) {
-        if (isElementOverridingStandaloneConfig(elements, position)) {
-            elements[position] = elements[position].replaceFirst(OVERRIDE_KEYWORD, "");
-        }
+    protected String getOrDefault(final CSVRecord record, int colNum, String defaultValue) {
+        return (record.get(colNum).isEmpty()) ? defaultValue : record.get(colNum).trim();
     }
 
     /**
-     * Gets the value of {@code position} in {@code elements}.
-     * Returns the value of {@code position} if it is in {@code element} and not empty.
-     * Otherwise returns an empty string.
+     * Returns the value of {@code record} at {@code colNum} as a {@code List},
+     * delimited by {@code COLUMN_VALUES_SEPARATOR} if it is in {@code record} and not empty, or
+     * returns an empty {@code List} otherwise.
      */
-    protected String getValueInElement(final String[] elements, int position) {
-        return (containsValueAtPosition(elements, position)) ? elements[position] : "";
+    protected List<String> getAsList(final CSVRecord record, int colNum) {
+        if (colNum >= record.size() || record.get(colNum).isEmpty()) {
+            return Collections.emptyList();
+        }
+        return Arrays.stream(record.get(colNum).split(COLUMN_VALUES_SEPARATOR))
+                .map(String::trim)
+                .collect(Collectors.toList());
     }
 
     /**
-     * Gets the value of {@code position} in {@code elements}.
-     * Returns the value of {@code position} if it is in {@code element} and not empty.
-     * Otherwise returns the {@code defaultValue}.
+     * Returns the values in {@code record} as a list with the {@link CsvParser#OVERRIDE_KEYWORD} prefix removed.
+     * Returns an empty list if {@code record} at {@code colNum} is empty.
      */
-    protected String getValueInElement(final String[] elements, int position, String defaultValue) {
-        return (containsValueAtPosition(elements, position)) ? elements[position] : defaultValue;
-    }
+    protected List<String> getAsListWithoutOverridePrefix(final CSVRecord record, int colNum) {
+        if (colNum >= record.size()) {
```

Comment:
Same comment as `containsValueAtPosition`.

Should have a method for checking.